app/controllers/sitemap_controller.rb

```diff
@@ -1,11 +1,16 @@
  class SitemapController < ApplicationController
 
    def index
-     @static_pages = [root_url, about_url, contact_url, faq_url, login_url, signup_url]
-     @frequently_updated_pages = [forum_url]
+     @static_pages = static_pages
      @courses = Course.all
```

Comment:
This is a very minor thing,
Can we keep the instance variables together please. I think it makes it look cleaner:
```
@static_pages = static_pages
@course = Course.all

respond_to do |format|
  format.xml
end
```